components/sidebar/pref_names.h

```diff
@@ -14,6 +14,7 @@ constexpr char kSidebarHiddenBuiltInItems[] =
 constexpr char kSidebarShowOption[] = "brave.sidebar.sidebar_show_option";
 constexpr char kSidebarItemAddedFeedbackBubbleShowCount[] =
     "brave.sidebar.item_added_feedback_bubble_shown_count";
+constexpr char kSidePanelWidth[] = "brave.sidebar.side_panel_width";
```

Comment:
Do we need to add this to the `pref_service_incognito_allowlist.cc`?

Reply:
Good catch. Added.